src/main/java/com/msalvatore/graphqlplayground/scalar/datetime/DateTimeHelper.java

```diff
@@ -0,0 +1,97 @@
+package com.msalvatore.graphqlplayground.scalar.datetime;
```

Comment:
I copied these classes wholesale from the starter. They could probably be cleaned up for my purposes, but it didn't seem worthwhile.